src/styles/general/_variables.scss

```diff
@@ -24,3 +28,4 @@ $space-s: 0.5rem; // 8px
 $space: 1rem; // 16px
 $space-m: 1.25rem; // 20px
 $space-l: 1.5rem; // 24px
+$space-xl: 3rem;
```

Comment:
Comment value in px here. This should be readable on first look, imagine if i just gave you project with just `rem` and let you calculate everything when you need to use it 😄

Reply:
@popovbojana You forgot about this